v1/api/swagger/paths/hosts@{hostId}.yaml

```diff
@@ -34,7 +34,8 @@ put:
   description: >-
     Updates the Host with the matching ID.  Update is permitted only if the host
     is in the 'Ready' or 'Connection Updating Failed' state.  Only the Host
-    'Description', 'Networks', and 'NetworkForDefaultRoute' can be updated.
+    'Description', 'Networks', 'NetworkForDefaultRoute', and 'ISCSIConfig:InitiatorName'
+    can be updated.
```

Comment:
document initiatorname update allowed only when no attached volumes?

Reply:
Since the 'ISCSIConfig:InitiatorName' update is possible, it must be marked as required in Host.yaml.
We can't differentiate whether the user intentionally wants to set it to "" or just didn't send it to request as it is not marked required.

Reply:
Good point. Will update.

Reply:
If somebody is using an old SDK, then also we will still have the same problem.

Reply:
As further discussed, there is no sure shot way to address access via an older SDK. Users of the go SDK will need to update the module. I will make the 'InitiatorName' required.